services/dis-apim-operator/config/samples/apim_v1alpha1_api.yaml

```
@@ -6,4 +6,47 @@ metadata:
+      displayName: Version 2
+      description: First version of the API
+      serviceUrl: https://example.com/v2
+      products:
+        - product1
+        - product2
+      protocols:
+        - https
+      subscriptionRequired: true
+      contentFormat: swagger-link-json
+      content: https://primary-test-aca-vga.gentleground-884783d5.norwayeast.azurecontainerapps.io/swagger/doc.json
+      isCurrent: true
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)